docker/Dockerfile

```diff
@@ -174,9 +174,12 @@ RUN addgroup -g ${GID} ${USER} && adduser -u ${UID} -D -G ${USER} -g "" ${USER}
 
 # Copy built minifi distribution from builder
 COPY --from=build --chown=${USER}:${USER} ${MINIFI_VERSIONED_HOME} ${MINIFI_HOME}
+RUN mkdir ${MINIFI_HOME}/scripts
+COPY --from=build --chown=${USER}:${USER} ${MINIFI_BASE_DIR}/docker/scripts/start.sh ${MINIFI_HOME}/scripts
+RUN chmod +x ${MINIFI_HOME}/scripts/start.sh
 
 USER ${USER}
 WORKDIR ${MINIFI_HOME}
 
 # Start MiNiFi CPP in the foreground
-CMD ["./bin/minifi.sh", "run"]
+CMD ["./scripts/start.sh"]
```

Comment:
Instead of this script, we should configure the minifi logger to output logs to stdout, through `minifi-log.properties`. Can we do that in the context of the docker image?

Reply:
stderr would be even better if docker and k8s is capable of using that.

Reply:
Good point, I checked with the NiFi team and it seems that they use this method due to legacy problems, but will be changed in the future, so we do not have any reason not to use stderr in a docker container by default for logging. I updated the default log properties in 38705267358e4e56f40a9a720dbcd68508a0a69f and we can use `docker logs` for every container in our test environment as well from now on.